src/layer/processing.ts

```diff
@@ -98,6 +98,13 @@ export function createProcessingPayload(
 ): ProcessingPayload {
   const { bbox } = params;
 
+  // Processing API supports setting multiple output responses, which are then returned as TAR archive.
+  // Sentinelhub-js can't deal with manipulating files inside the tar yet,
+  // so we only allow setting one output response id.
+  if (params.outputResponseId === '') {
+    throw new Error('outputResponseId most not be empty');
+  }
+
```

Comment:
Unless I'm missing something, the comment doesn't match the code? 

Also, the code handles a very specific case where `params.outputResponseId === ''`, which might not need to be handled (it can be set to `"default"` lower). It is a very specific corner case which will probably never arise, and if it does, one could argue that  using `"default"` is what the user would have wanted to do anyway. If you agree, I would suggest removing these lined altogether (though the comment might be useful if put elsewhere - `src/layer/const.ts` line 35 maybe?).

Very minor though, fftr either way.

Reply:
Thanks, kind of forgot to move the comment to a better place - moved it now to line 35 in `src/layer/const.ts` - much better location.
I also removed the check as suggested. Setting the empty string will hopefully never happen, but if it does, we can explain / change the behaviour then.

Reply:
Resolving